dataland-community-manager/src/main/resources/application.properties

```diff
@@ -26,8 +26,8 @@ spring.datasource.hikari.maximum-pool-size=50
 mailjet.api.url=https://api.eu.mailjet.com
 dataland.notification.sender.address=info@dataland.com
 dataland.notification.sender.name=Dataland
-dataland.notification.receivers.bulk.data.request=${NOTIFICATION_RECEIVERS_BULK_DATA_REQUEST:dummy.receiver@dataland.com}
-dataland.notification.receivers.cc.bulk.data.request=${NOTIFICATION_RECEIVERS_CC_BULK_DATA_REQUEST:dummy.receiver2@dataland.com}
+dataland.notification.bulk-data-request.receivers=${NOTIFICATION_RECEIVERS_BULK_DATA_REQUEST:dummy.receiver@dataland.com}
```

Comment:
Why this change?

Reply:
Because it made no structural sense before. `.`s should not be used as separators for names. And receivers are the property. Not bulk-data-request a property of receivers.

Reply:
ok, makes sense